config/config.exs

```diff
@@ -31,6 +31,27 @@ config :animina, :ai_message_help_price, 20
 # Configures whether to have ai automatically respond to messages
 config :animina, :autoreply_messages_with_ai, false
 
+# Configures default maximum age for potential partners
+config :animina, :default_potential_partner_maximum_age, 100
+
+# Configures default minimum age for potential partners
+config :animina, :default_potential_partner_minimum_age, 18
+
+# confugures the difference in age if no maximum or minimum age is set
+config :animina, :default_potential_partner_age_difference, 10
+
+# Configures the maximum height for potential partners
+config :animina, :default_potential_partner_maximum_height, 300
```

Comment:
300 as a default doesn't make any sense. Make it 120 please.

PS: There will come the day when I get an email from a user who has a height of 121 cm. I am so sure! ;-)

Reply:
In this case we want a value so huge that will make sure we do not leave out any users , we also use this value as the default maximum height in the query , this allows us to select all users below 300 cm , I think it would be best to have a very high value for this case , thoughts? @wintermeyer

Reply:
A quick google research let to the number 251. That seems to be the height human right now. Let's go with 225 cm since the heightest German right now seems to be 224.